components/clarity-repl/src/repl/session.rs

```diff
@@ -836,6 +840,14 @@ impl Session {
             "{}",
             help_colour.paint("::read <filename>\t\t\tRead expressions from a file")
         ));
+        output.push(format!(
+            "{}",
+            help_colour.paint("::encode <expr>\t\t\t\tEncode an expression to a Clarity Value bytes representation")
+        ));
+        output.push(format!(
+            "{}",
+            help_colour.paint("::decode <expr>\t\t\t\tDecode a Clarity Value")
```

Comment:
```suggestion
            help_colour.paint("::decode <bytes>\t\t\t\tDecode a Clarity Value bytes representation")
```

Could we make it easier to understand what is expected here?

Reply:
Sure... better description indeed. Fixed.